packages/rulesets/src/native/rulesets/arm.ts

```diff
@@ -14,20 +14,35 @@ import { providerNamespace } from "../functions/provider-namespace"
 export const armRuleset: IRuleSet = {
   documentationUrl: "https://github.com/Azure/azure-openapi-validator/blob/develop/docs/rules.md",
   rules: {
-
     ///
     /// ARM RPC rules for Delete patterns
     ///
 
     // https://github.com/Azure/azure-openapi-validator/issues/329
     // RPC Code: RPC-Delete-V1-03
-    AllResourcesMustHaveDelete: {
+    AllTrackedResourcesMustHaveDelete: {
       category: "ARMViolation",
       openapiType: OpenApiTypes.arm,
       severity: "error",
       given: "$",
       then: {
         execute: allResourcesHaveDelete,
+        options: {
+          isTrackedResource: true,
+        },
+      },
+    },
+    // RPC Code: RPC-Delete-V1-03
```

Comment:
>RPC-Delete-V1-03 [](http://example.com/codeflow?start=17&length=16)

RPC-Delete-V1-05

Reply:
sorry, I thought we were just refactoring RPC-Delete-V1-03 into 2 different rules
Updated.